### ISSUE TITLE
Delete person

### ISSUE THREAD

**Issue by djbusstop** [zetkin/app.zetkin.org#558]
### Description
A user should be able to delete a person record entirely

### Relevant Job Stories
- None

### Prerequisites
#554 

### Requirements
- [ ] WIP

### Open questions
- Refer to this question: https://github.com/zetkin/app.zetkin.org/issues/452#issuecomment-1027899340

**Comment by richardolsson** [comment]
Following up on the discussion from #452 with @djbusstop, @GemCopeland and @lowlandjuju.

There are actually two ways that a person record can be "deleted". Imagine the following organization tree, which the person p is connected to four organizations:

```
       A (P)
   ____|____
  |         |
  B (P)     C (P)
        ____|____
       |         |
       D         E (P)

```

This approximates a quite typical organization tree consisting of national, regional and local levels.

Maybe an official of organization E has imported a list that includes P, and through the process of what we call "membership propagation" that person is also added to C and A. And then the physical individual P has created a user account in Zetkin and explicitly connected to organization B.

Knowing this, we have previously envisioned two different concepts of "delete":

## 1. "Disconnecting"
Disconnecting P from an organization will only remove it from lists in that organization (and it's sub-organizations). For example, if an official in C deletes P this way, P will disappear from C and E only, but remain in A and B as if nothing happened.

Use case: Someone has moved away from my local branch I don't want to organize them anymore, but is still a member of the overarching organization and organized elsewhere.

## 2. "Deleting"
Deleting P entirely will flag it for deletion and hide it everywhere as if P never existed. P will disappear from lists in A, B, C and E, i.e. all organizations it was ever in.

Use case: Someone has decided to quit the organization completely, and communicated that to me, an official, so I want to delete them entirely from Zetkin.

How do we properly communicate these two options to the organizer?

**Comment by richardolsson** [comment]
Thinking back, I think maybe @GemCopeland intended for the separate card containing organizations (#555) to be the way to remove a person from the current organization as well as sub-organizations?

**Comment by lowlandjuju** [comment]
Revisiting this - "disconnection" is now handled by the merged PR implementing #555 
So, "deletion" should IMHO be a big red button on a "manage" tab on the profile page.
